### PR TITLE
Home page update

### PR DESCRIPTION
**Pull request by carpenterd777**
- Add recordings page
- Move contributions to own page
- add styling to contributions view
- allow article list to grow freely

These changes aim to clean up the home page. Old material that is no longer relevant to the current semester long project is moved from the home page to other locations in the website, such as pages accessible in the "more" sidebar. The scrolling feature of the article list was removed to enhance the feel of navigation but not having to deal with nested scroll bars.

### CODE REVIEW

**Comment by dyc3** [comment]
FYI, the time in the flyer and the time shown on the website for GBM 1 is different

File: src/views/Contributions/Contributions.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import * as SEC from "../../components/SEC/lib.js";
+import Contributions from "../../components/Contributions/Contributions";
+import "./Contributions.scss";
+
+const ContributionsView = () => {
+	return (
+		<div className="content">
+			<h1>Open Source Contributions</h1>
+			<div>
+				<SEC.Link href="https://github.com/orgs/StevensSEC/projects/3">
```
Comment (dyc3) [inline src/views/Contributions/Contributions.js:12]: this is not updated

**Review by dyc3 — CHANGES_REQUESTED** [review]
Recordings and Contributions pages don't have great alignment

image

image

**Comment by carpenterd777** [comment]
> 
> 
> FYI, the time in the flyer and the time shown on the website for GBM 1 is different

yeahhhhh I'm trying to get that fixed now in #286

**Review by dyc3 — APPROVED** [review]

**Pull request merged**